File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BaseAddressType.vb
Public Class BaseAddressType
    Public Property AddressKey As String
    Public Property AddressLine1 As String
    Public Property AddressLine2 As String
    Public Property AddressLine3 As String
    Public Property AddressLine4 As String
    Public Property AddressLine5 As String
    Public Property AddressLine6 As String
    Public Property AddressLine7 As String
    Public Property AddressLine8 As String
    Public Property AddressLine9 As String
    Public Property AddressLine10 As String
    Public Property AddressTypeCode As AddressTypeType = AddressTypeType.Correspondence
    Public Property CountryCode As String
    Public Property PostCode As String
    Public Property CountryId As Integer?
End Class

File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BaseAddressType.cs
using Newtonsoft.Json;
using SSP.PureInsuranceRestAPIHandler.Enums;
using System;

namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BaseAddressType
    {
        public string AddressKey { get; set; }

        public string AddressLine1 { get; set; }

        public string AddressLine10 { get; set; }

        public string AddressLine2 { get; set; }

        public string AddressLine3 { get; set; }

        public string AddressLine4 { get; set; }

        public string AddressLine5 { get; set; }

        public string AddressLine6 { get; set; }

        public string AddressLine7 { get; set; }

        public string AddressLine8 { get; set; }

        public string AddressLine9 { get; set; }
        public AddressTypeType? AddressTypeCode { get; set; } = AddressTypeType.Correspondence;
        
        [JsonProperty(Required = Required.AllowNull)]
        public string CountryCode { get; set; } 
        [JsonProperty(Required = Required.AllowNull)]
        public string PostCode { get; set; }
        [JsonProperty(Required = Required.AllowNull)]
        public Nullable<int> CountryId { get; set; } = 0;
    }
}
